addons/hr_expense/models/hr_expense.py

```diff
@@ -847,16 +847,37 @@ def _prepare_payments_vals(self):
                 for attachment in self.message_main_attachment_id]
         }
 
+    def _get_base_account(self):
+        # We need to do this as the installation process may delete the original account, and it doesn't recompute properly after.
```

Comment:
If the comment is there to explain the function it's better to use a docstring instead of a single #